.github/configs/cluster-config.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: kind.x-k8s.io/v1alpha4
+kind: Cluster
+nodes:
+  - role: control-plane
+    kubeadmConfigPatches:
```

Comment:
In my testing I found that, in addition to the configmap patches like you have here, I also had to modify the manifests to reflect these changes and add the port config too, e.g. for controller manager `/etc/kubernetes/manifests/kube-controller-manager.yaml`:

```
ports:
- containerPort: 10257
  name: https-metrics
  protocol: TCP
```

Not sure if this `kubeadmConfigPatches` handles that already?